readme.md

```diff
@@ -18,7 +18,8 @@ var BrowserStack = require( "browserstack" );
 // REST API
 var client = BrowserStack.createClient({
 	username: "foo",
-	password: "p455w0rd!!1"
+	password: "p455w0rd!!1",
+	proxy (optional): "http://myproxy.com:1234"
```

Comment:
This should be copy-pastable code and the parenthesis would throw a syntax exception... I'd suggest replacing it by 

`proxy: "http://myproxy.com:1234" // proxy configuration is optional`

Or even not stating it at all in the docs, I guess it is not such a common use case.